### DISCUSSION TITLE
Covenant Abilities merged with Cooldowns

### DISCUSSION THREAD

**Discussion by Bigity** [Hekili/hekili#1112] [Q&A]
Is there a way to combine covenant abilities with regular cooldowns for the purposes of display and recommendation?

So I can toggle them both on/off with the same key press?

**Comment by Hekili** [comment]
There's a checkbox that makes the Cooldowns toggle override the Covenants toggle in /hekili > Toggles.

That doesn't apply to which display the abilities appear in, however.  You'd need to swap them from being on the Covenants toggle to the Cooldowns toggle via /hekili > Abilities.

**Reply by Bigity** [reply]
Thanks!
